README.md

```diff
@@ -61,56 +63,65 @@ pip install noteable-origami
 
 ## Getting Started
```

Comment:
A note here with big warning emojis should say "This documentation is written for the alpha release. For the stable release, see [link to commit]."